LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-invalid-expected.txt

```diff
@@ -35,7 +35,7 @@ PASS e.style['opacity'] = "pow(1, , 2)" should not set the property value
 PASS e.style['letter-spacing'] = "calc(1px * pow(1))" should not set the property value
 PASS e.style['letter-spacing'] = "calc(1px * pow(2px, 3px))" should not set the property value
 PASS e.style['letter-spacing'] = "calc(sqrt(100px)" should not set the property value
-PASS e.style['letter-spacing'] = "hypot(2px, 40%)" should not set the property value
+FAIL e.style['letter-spacing'] = "hypot(2px, 40%)" should not set the property value assert_equals: expected "" but got "hypot(2px, 40%)"
```

Comment:
I think we should change the WPT if it's incorrect. I do indeed see no reason for this to be disallowed.

Reply:
I'm pretty sure WebKit is wrong on this one, not the test. letter-spacing is supposed to "normal | <length>", but we have it defined to "normal | <length-percentage>". 

I am not sure why webkit has the definition the way it does, but it doesn't seem like the right change to make in this PR.